README.md

```diff
@@ -21,27 +48,19 @@ You can also find a set of smaller feature-focused samples in the [Runtime Sampl
 * [`Editing geometries in a SceneView`](src/SceneViewEdit) - A Windows Desktop app demonstrating how to do geometry editing in 3D using a customer editor.
 * [`Measure Tool`](src/MeasureTool) - A simple sample demonstrating how to use the editor to create your own measure tool.
```

Comment:
Not part of this PR and really not a huge thing, but I think the demo app descriptions could be made a bit more consistent. Many of these use the format "A --- that [demonstrates/shows/illustrates] ...". I think it would be good to use that format throughout.